content-scripts/splitgraph-docs/htmlToMdx.js

```diff
@@ -27,7 +27,7 @@ const preprocessHtml = (html) => {
 };
 
 const postprocessHtml = (html) => {
-  return html.replace(/\sclass=\"/gm, ' className="');
+  return html.replace(/\sclass="/gm, ' className="');
```

Comment:
These changes made me pretty nervous, so I ran the `yarn update-docs` script and looked closely to make sure the `htmlToMdx` was correctly replacing `class="` with `className="`. It looks correct to me.

I guess this implies JS ignores overzealous escaping (which makes sense given you need to double-escape a backslash to match for one). Good ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#escaping

(in the course of checking this, I did identify a different bug in the updateDocs script that's unrelated to this PR – I'll push a separate fix)

Reply:
I trusted ESLint and the suggestions seemed sensible to me :smile: 

I'm glad that contributed to a possible bugfix after all!